MapboxSearchUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MapboxSearchUI'
-  s.version          = '1.0.0-rc.2'
+  s.version          = '1.0.0-rc.3'
   s.summary          = 'Search UI for Mapbox Search API'
```

Comment:
Let's increase version for the `SearchSDK` dependency here as well.
Here is an [example]( https://github.com/mapbox/mapbox-search-ios/pull/103/files#diff-50662b0f63d56eca9ff6b9c56939241dae4c8c679c41c7edab670be8d6424ad7).